Podcast/Slider.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Slider.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 11/13/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class Slider: UISlider {
+
+    let tapWidth: CGFloat = 44
+    let tapHeight: CGFloat = 44
+    
+    init() {
+        super.init(frame: .zero)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func point(inside point: CGPoint, with event: UIEvent?) -> Bool {
+        let area = bounds.insetBy(dx: bounds.width < tapWidth ? bounds.width - tapWidth : 0, dy: bounds.height < tapHeight ? bounds.height - tapHeight : 0)
```

Comment:
Did we need a subclass for this? Was there not a way to increase the button size of our current slider? Or is our image just too small so this was easiest?

Reply:
i got eileen to increase the button size of the slider by a little bit but it's still a bit small, and having the slider too big also looks pretty ugly so i thought this would be a good compromise